lib/cocoapods/command/repo/push.rb

```diff
@@ -39,6 +41,7 @@ def initialize(argv)
           @podspec = argv.shift_argument
           @use_frameworks = !argv.flag?('use-libraries')
           @private = argv.flag?('private', true)
+          @message = argv.flag?('message')
```

Comment:
That crossed my mind and it certainly would be less complex. However, the reason for implementing this feature is to enable contributors to write a summary of what's changed in the version that's pushed to the specs repo. This is especially useful for private specs repos as the commit history of the specs repo then acts as a convenient changelog. This works very good in combination with, for example, Slack integration. Team members are automatically updated about changes of internal libraries.